make/production/jumpstart-academic.make

```diff
@@ -1,5 +1,7 @@
 core = 7.x
 api = 2
+; Default directory should be sites/all/modules/contrib
+defaults[projects][subdir] = contrib
```

Comment:
I can't remember why at this point in time but I had a reason to not do this in the Deployer make files. Let me see if I can shake out the cobwebs.